build.gradle

```diff
@@ -0,0 +1,191 @@
+buildscript {
+  ext.junitJupiterVersion = '5.6.1'
+  ext.pegasusVersion = '29.22.16'
+  ext.mavenVersion = '3.6.3'
+  ext.springVersion = '5.3.19'
+  ext.springBootVersion = '2.5.12'
+  apply from: './repositories.gradle'
+  buildscript.repositories.addAll(project.repositories)
+  repositories {
+    mavenCentral()
+    mavenLocal()
+    maven {
+      url "https://repository.mulesoft.org/nexus/content/repositories/public/"
+    }
+    maven {
+      url "https://linkedin.jfrog.io/artifactory/open-source/" // GMA, pegasus
+    }
+
+  }
+  dependencies {
+    classpath 'com.linkedin.pegasus:gradle-plugins:' + pegasusVersion
+  }
+}
+
+plugins {
+  id 'java'
+  // Currently "maven-publish" has some issues with publishing to Nexus repo. So, we will use a different plugin.
+  // See https://issues.sonatype.org/browse/OSSRH-86507 for more details.
+  id "com.vanniktech.maven.publish" version "0.22.0"
+  id 'signing'
+  id("io.github.gradle-nexus.publish-plugin") version "1.1.0"
+}
+
+repositories {
+    mavenCentral()
+    mavenLocal()
+    maven {
+      url "https://repository.mulesoft.org/nexus/content/repositories/public/"
+    }
+    maven {
+      url "https://linkedin.jfrog.io/artifactory/open-source/" // GMA, pegasus
+    }
+
+}
+
+configurations {
+  // configuration that holds jars to include in the jar
+  extraLibs
+
+  // Dependencies that will be provided at runtime in the cloud execution
+  provided
+
+  compileOnly.extendsFrom(provided)
+  testImplementation.extendsFrom provided
+}
+
+jar {
+  archivesBaseName = "feathr_2.12"
+  duplicatesStrategy = DuplicatesStrategy.EXCLUDE
+  manifest {
+    attributes(
+        'Class-Path': [project.configurations.runtimeClasspath],
+        'Main-Class': 'com.linkedin.feathr.offline.job.FeatureJoinJob',
+        "Implementation-title": "Build jar for local experimentation"
+    )
+  }
+  from {
+    configurations.runtimeClasspath.collect { it.isDirectory() ? it : zipTree(it) }
+  }
+}
+
+
+group = "com.linkedin.feathr"
+version = "0.10.3"
```

Comment:
Move to top

Reply:
Merge with version=0.10.3-rc4 in gradle.properties?

Reply:
Updated.